src/main/java/com/blackducksoftware/integration/hub/alert/web/AuthenticationHandler.java

```diff
@@ -34,6 +34,8 @@
     @Override
     protected void configure(final HttpSecurity http) throws Exception {
         http.csrf().disable().authorizeRequests().anyRequest().permitAll().and().logout().logoutSuccessUrl("/");
+        // TODO: Make sure the H2 console is disabled by removing the following line of code.
```

Comment:
Is this being committed for debugging purposes?

Reply:
This is for debugging purposes as we develop our initial version.  Before release this will need to be disabled.